app/models/search/indexer.rb

```diff
@@ -51,6 +63,16 @@ def self.create_index(shards = 5)
     )
   end
 
+  def self.prepare_for_testing
+    raise "Wrong environment for test prep!" unless Rails.env.test?
+    delete_index
+    create_index
+  end
+
+  def self.refresh_index
```

Comment:
We only do this for tests, should we add a guard clause like `prepare_for_testing`?